.github/workflows/ci.yaml

```diff
@@ -66,6 +66,31 @@ jobs:
           path: report.*
           if-no-files-found: ignore
 
+  check-dock:
+    name: Check c-lightning doc
+    runs-on: ubuntu-20.04
+    env:
+      DEVELOPER: 1
+      VALGRIND: 0
+      EXPERIMENTAL_FEATURES: 0
+      COMPAT: 1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.0.0
+
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+
+      - name: Install dependencies
+        run: bash -x .github/scripts/setup.sh
+
+      - name: Check Manpages
+        run: |
```

Comment:
Add a line `pip3 install mako` here

Reply:
I meant right after `run: |`, but could be elsewhere maybe. Didn't test it.

Reply:
No, that'd fail in the `NO_PYTHON` configuration which is run on CI. Incidentally this should rather be fixed by checking in the `channel_type_wiregen` files in #4794